app/LengthConvertor.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace ConvertorApp;
+class LengthConvertor {
+	/*
+	 *This function Converts the Input Length from Meter To Kilometer
+	 */
+	public function convertMeterToKilometer($length) {
+		if (is_string($length)) {
+			throw new \InvalidArgumentException("Please Enter valid Length");
+		} elseif (is_null($length)) {
+			throw new \InvalidArgumentException("Please Enter valid Length");
+		}
+		$KiloMeter = $length/1000;
```

Comment:
follow camel case. `$KiloMeter` is variable it should be named `$kiloMeter` or `$kilometer`